core/embed/rust/src/ui/model_tr/component/show_more.rs

```diff
@@ -0,0 +1,98 @@
+use crate::ui::{
+    component::{Child, Component, Event, EventCtx},
+    geometry::{Insets, Rect},
+};
+
+use super::{theme, ButtonController, ButtonControllerMsg, ButtonLayout, ButtonPos};
+
+pub enum CancelInfoConfirmMsg {
+    Cancelled,
+    Info,
+    Confirmed,
+}
+
+pub struct ShowMore<T> {
+    content: Child<T>,
+    buttons: Child<ButtonController>,
+}
+
+impl<T> ShowMore<T>
+where
+    T: Component,
+{
+    pub fn new(content: T) -> Self {
+        let btn_layout = ButtonLayout::cancel_armed_text("CONFIRM".into(), "ALL".into());
```

Comment:
Please change "ALL" to "i".

Reply:
Done in [0c1e598](https://github.com/trezor/trezor-firmware/pull/2610/commits/0c1e598f90a73bbf235a738477ad7e25a6d53794)